test/cli/install/registry/packages/bunx-bins/package.json

```diff
@@ -43,7 +44,7 @@
 	"_distfiles": {},
 	"_attachments": {
 		"bunx-bins-1.0.0.tgz": {
-			"shasum": "c23382ce43a04a94035050ac736f04179a8639ff",
+			"shasum": "a77e200b47158585a4bf70ef934202c5fb2b0dd7",
 			"version": "1.0.0"
 		}
```

Comment:
☝️ Is this produced from a pack'd npm/bun package? Are there documented steps with how to update this test when needed?

Reply:
npm publish